src/components/dashboard/ScsInProgressDialog/ScsDialog.js

```diff
@@ -158,16 +159,45 @@ const ScsInProgressDialog = ({ classes, intl, scsInProgress }) => {
     setScStatus([]);
   };
 
+  let label;
+  switch (status) {
```

Comment:
You could you shorter solutions:

`const labelId = {
    INITIALIZATION: 'dashboard.initialization',
    IN_PROGRESS: 'dashboard.inprogress',
    READY: 'dashboard.ready',
    CLOSED: 'dashboard.closed',
    ARCHIVED: 'dashboard.archived'
  }` 
And then use this labelId in translateGeneralStatus InfoWidget'label property, where you simply create int. object.

But we already have such method in **string.js:**

`export const translateGeneralStatus = status => {
  switch (status) {
    case 'INITIALIZATION':
      return 'sc.phase.preparation';
    case 'IN_PROGRESS':
      return 'sc.phase.inProgress';
    case 'READY_TO_CLOSE':
      return 'sc.phase.ready';
    case 'DONE':
      return 'sc.phase.closed';
    case 'ARCHIVED':
      return 'sc.phase.archived';
    default:
      return 'Unbekannt';
  }
};`
and these are translated also in english. Just check if the words are as desired.